recipes/hapcut2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "HapCUT2" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/vibansal/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: f9fb1838b4c3f7fa4dc7bd6de72341f950caa3b2cf98b40a73927734dc75e26e
+    patches:
+    - shebang.patch
+    - flags.patch
+    - segfault.patch  # Patch from upstream master that’s not in the 1.1 release
+  - url: https://github.com/samtools/htslib/releases/download/1.2.1/htslib-1.2.1.tar.bz2
+    sha256: bddb1372b9215b32e0f21d8394bab859c57cb00324a8dc06775cdac5b4d662b0
+    folder: submodules/htslib
+  - url: https://github.com/samtools/samtools/releases/download/1.2/samtools-1.2.tar.bz2
+    sha256: 420e7a4a107fe37619b9d300b6379452eb8eb04a4a9b65c3ec69de82ccc26daa
+    folder: submodules/samtools
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - zlib
+  run:
+    - zlib
+    - python
+    - pysam
+
+build:
```

Comment:
`build` should be above requirements, usually ... :)

Reply:
Fixed now :)